drive_cli/dcli.py

```diff
@@ -25,46 +25,68 @@
 
 
 def login(remote):
-    token = os.path.join(dirpath, 'token.json')
-    store = file.Storage(token)
-    creds = store.get()
-    if not creds or creds.invalid:
-        client_id = os.path.join(dirpath, 'oauth.json')
-        flow = client.flow_from_clientsecrets(client_id, SCOPES)
-        flags = tools.argparser.parse_args(args=[])
-        if remote:
-            flags.noauth_local_webserver = True
-        creds = tools.run_flow(flow, store, flags)
-        click.secho(
-            "********************** welcome to **********************", bold=True, fg='red')
-        result = pyfiglet.figlet_format("Drive - CLI", font="slant")
-        click.secho(result, fg='yellow')
-        click.secho(
-            "********************************************************", bold=True, fg='red')
+    try:
+        token = os.path.join(dirpath, 'token.json')
```

Comment:
Hi, the exception handling can be better by reducing the lines under `try` block. You don't need to try everything and it becomes a little difficult for the developer to figure out where the exception might happen.